### PR TITLE
chore: most recent code review changes

### PR DESCRIPTION
**Pull request by bcoe**
upstream changes from most recent code review.

### CODE REVIEW

**Comment by shadowspawn** [comment]
Added Ian's suggestions.

**Review by shadowspawn — APPROVED** [review]
LGTM. Verified changes against review comments.

**Comment by shadowspawn** [comment]
Removed stale `type` default of `boolean` in API docs, which has already been applied upstream.

File: index.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
   ArrayPrototypeSlice,
   ArrayPrototypePush,
+  ArrayPrototypePushApply,
   ObjectEntries,
   ObjectPrototypeHasOwnProperty: ObjectHasOwn,
   StringPrototypeCharAt,
   StringPrototypeIncludes,
   StringPrototypeIndexOf,
   StringPrototypeSlice,
+  ArrayPrototypeUnshiftApply
```
[the remark this answers is not in the document]
Comment (shadowspawn) [reply index.js:15]: They are upstream.

[the remark this answers is not in the document]
Comment (shadowspawn) [reply index.js]: Reordered!

**Review by Eomm — APPROVED** [review]

**Comment by shadowspawn** [comment]
I did a file compare against matching upstream files for `index.js`, `utils.js`, and `README.md`. Just picked up one non-whitespace difference and added a suggestion on the PR to catch up:
- https://github.com/nodejs/node/pull/42675/files#r862629844

**Pull request merged**